src/v2/guide/accessibility.md

```diff
@@ -0,0 +1,301 @@
+
+- Keyboard-only users won't be able to move focus to the button, and so can't click on it to toggle it. You can test this yourself by attempting to tab onto the element (start in the input elements at the beginning of this section).
+- The element is completely invisible to screen readers, so not only will people using screen readers not be able to change the value of the button, but they won't even know that it is there.
+
+The first problem is fairly easy to solve: you can just add `tabindex="0"` to the element and you can then tab to it. Note in the next demo that the element is given an outline when focused: this is good! It means that users who aren't using a mouse can tell where the focus is.
+
+The second isn't as simple, and there are two ways to solve it:
+
+#### Extend a native input element
+
+The first possible way to solve this problem is to build the component by starting with a native input element, hiding it from displaying, and making our new HTML hidden from screen readers.
+
+For example, with the previous demo, we could have a hidden checkbox input bound to the same variable, like this:
+
+```html
+<input v-model="checked" class="sr-only" type="checkbox">
+<div
+  aria-hidden="true"
+  tabindex="0"
+  :class="['toggle-button', { 'toggle-button--checked': checked }]"
+  @click="checked = !checked">
+  <div class="toggle-button__handle"></div>
+</div>
+```
+
+{% raw %}
+<div id="example-2" class="demo">
+  <input v-model="checked" class="sr-only" type="checkbox">
+  <div
+    aria-hidden="true"
+    tabindex="0"
+    :class="['toggle-button', { 'toggle-button--checked': checked }]"
+    @click="checked = !checked">
+    <div class="toggle-button__handle"></div>
+  </div>
+  
+  <pre>{ checked: {{ checked }} }</pre>
+</div>
+<script>
+var example1 = new Vue({
+  el: '#example-2',
+  data: {
+    checked: false,
+  },
+})
+</script>
+{% endraw %}
+
+The `sr-only` class means that the checkbox is visible only to screen readers, and `aria-hidden="true"` on the toggle button means that it is hidden to screen readers. Screen readers read out the checkbox, and your sighted users see the toggle button as intended—it works for both parties.
+
+#### ARIA attributes
+
+We saw an ARIA (short for [Accessible Rich Internet Applications][WAI-ARIA]) attribute in the previous example to hide the custom HTML from the screen reader. aria-hidden is one of a tonne of attributes that you can add to elements to tell assistive technology such as screen readers what things on the page are.
+
+Check out MDN for a longer read on ARIA: [An overview of accessible web applications and widgets][ARIA MDN].
+
+Additionally, the W3C keeps a detailed guide of usage and examples for ARIA: [Using Aria - W3C Working Draft][Using ARIA]. This also includes a handy list of [ARIA roles][ARIA Roles] and [ARIA states and properties (or attributes)][ARIA States and Properties].
+
```

Comment:
I highly recommend adding the [W3C aria practices](https://www.w3.org/TR/wai-aria-practices-1.1/) page to guide developers to learn and follow. Because it supplies so many code cases that we can just copy-paste to start. Actually that's my very beginning way about supporting a11y in my personal project.